experimental/Schemes/MorphismFromRationalFunctions.jl

```diff
@@ -122,7 +122,9 @@ function Base.show(io::IOContext, Phi::MorphismFromRationalFunctions)
   if get(io, :supercompact, false)
     print("Morphism from rational functions")
   else
-    print("Hom: ", X, " -> ", Y)
+    io = pretty(io)
+    print(io, "Hom: ")
+    print(io, Lowercase(), domain(Phi), " -> ", Lowercase(), codomain(Phi))
```

Comment:
In other functions we print this in "supercompact" mode. But not here and not in some others. One can argue either way, for now I wanted to keep this change "simple" while also aligning it as much as possible with other `show` methods for maps here and in other repos.

Reply:
I do not understand, which part is printed in "supercompact" in other functions ?

Reply:
This is "one line" printing mode, and the recommendation (though not a requirement) is that any delegation it performs uses "supercompact" mode, to help ensure the "one line" promise is kept. So for other kinds of maps we do
```
print(IOContext(io, :supercompact => true), Lowercase(), domain(x), " -> ", Lowercase(), codomain(x))
```